common/src/main/java/com/tc/properties/TCPropertiesConsts.java

```diff
@@ -407,4 +387,15 @@
   public static final String  LOGGING_CATEGORY                                              = "logging";
   public static final String  NETCORE_CATEGORY                                              = "net.core";
 
+  String[] TC_PROPERTIES_WITH_NO_DEFAULTS = {
+      ENTITY_PROCESSOR_THREADS,
+      L2_TCCOM_WORKERTHREADS,
+      L2_SEDA_STAGE_WORKERTHREADS,
+      L2_NHA_TCGROUPCOMM_RECONNECT_L2PROXY_TO_PORT,
+      CLIENT_MAX_PENDING_REQUESTS,
```

Comment:
looks like some of these properties can be moved to tc.properties, can be fixed later though.